ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -518,4 +518,125 @@ if(displayMap) {
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>long|short|default|forbid</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </cache>
+      <code>
+        <contenttype>PureText</contenttype>
+        <disabled>0</disabled>
+        <editor>PureText</editor>
+        <name>code</name>
+        <number>2</number>
+        <prettyName>Code</prettyName>
+        <rows>20</rows>
+        <size>50</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </code>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>1</number>
+        <prettyName>Name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <parse>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>parse</name>
+        <number>4</number>
+        <prettyName>Parse content</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </parse>
+      <use>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <multiSelect>0</multiSelect>
+        <name>use</name>
+        <number>3</number>
+        <prettyName>Use this extension</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>currentPage|onDemand|always</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </use>
+    </class>
+    <property>
+      <cache>long</cache>
+    </property>
+    <property>
+      <code>require(['jquery'], function ($) {
+  var convertStringToDate = function(dateStr) {
+    var dateParts = dateStr.split("/");
+    var timeParts = dateParts[2].split(" ")[1].split(":");
+    dateParts[2] = dateParts[2].split(" ")[0];
+    return new Date(dateParts[2], dateParts[1] - 1, dateParts[0], timeParts[0], timeParts[1]);
+  }
+  var compareDates = function(startDate, endDate) {
+    var saveButton = $('input[name*=action_save]');
+    var sDate = convertStringToDate(startDate);
+    var eDate = convertStringToDate(endDate);
+    if (sDate &gt;= eDate) {
+      saveButton.prop('disabled', true);
+      new XWiki.widgets.Notification(
+        $jsontool.serialize($services.localization.render('contrib.meeting.message.invalidDate')), 'error');
+    } else {
+      saveButton.prop('disabled', false);
+    }
+  };
+  var startDate = $('#Meeting\\.Code\\.MeetingClass_0_startDate');
+  var endDate = $('#Meeting\\.Code\\.MeetingClass_0_endDate');
+  startDate.change(function() {
```

Comment:
I would add a comment about why you do this

Reply:
You can also remove the duplication with:

    startDate.add(endDate).change(...);